inference-engine/src/mkldnn_plugin/nodes/gather.cpp

```diff
@@ -22,18 +22,41 @@ namespace InferenceEngine {
 namespace Extensions {
 namespace Cpu {
 
+using MKLDNNPlugin::TensorDescCreatorTypes;
+
 class GatherImpl: public ExtLayerBase {
 public:
+    static bool isSupportedParams(const std::shared_ptr<ngraph::Node>& op, std::string& errorMessage) noexcept {
```

Comment:
I just wondering if we could use something similar to std::optional here that would not require the user to prepare an empty string for the message hadling and, moreover would not require  the function to write into external memory.

Reply:
Discussed offline. Decided to keep as is for now.